server/core_storage.go

```diff
@@ -607,7 +607,7 @@ func storageWriteObject(ctx context.Context, logger *zap.Logger, tx *sql.Tx, aut
 		}
 		return nil, err
 	}
-	if rowsAffected, _ := res.RowsAffected(); rowsAffected != 1 {
+	if rowsAffected, err := res.RowsAffected(); rowsAffected != 1 {
```

Comment:
I'm not really sure if `rowsAffected != 1` returns an error but I think it won't hurt to try logging it.